docs/fundamentals/syslib-diagnostics/syslib1090-1099.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: SYSLIB diagnostics for Microsoft.Interop.COMInteropGenerator
+title: SYSLIB diagnostics for Microsoft.Interop.COMInteropGenerator in .NET 8 and later
```

Comment:
It's unfortunate that these can't all be in the same file (1090-1099 and 1230-1239). It seems like they should be. Right? It's just the TOC entries that we'd need to figure out.

Reply:
Yeah I don't like them being in separate files. I was just following the current pattern. I got lucky that we ended up splitting the diagnostic groups by release version.

I'm fine changing this to however you'd like to make it one file; I just didn't want to make assumptions about how you'd want to do it.